### PR TITLE
Improved tui and use gopenqa

### PR DESCRIPTION
**Pull request by grisu48**
This PR contains three commits, which change the internals to use the [gopenqa](https://github.com/grisu48/gopenqa) library and we also separate the `tui` module of `openqa-mon` from the core internals of the tool.

### CODE REVIEW

**Comment by grisu48** [comment]
Preview: [openqa-mon.zip](https://github.com/grisu48/openqa-mon/files/6062644/openqa-mon.zip)

**Review by pdostal — APPROVED** [review]
From the user point of view I like it!

File: cmd/openqa-mon/tui.go [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+const ANSI_YELLOW = "\u001b[33m"
+const ANSI_BRIGHTYELLOW = "\u001b[33;1m"
+const ANSI_BLUE = "\u001b[34m"
+const ANSI_MAGENTA = "\u001b[35m"
+const ANSI_CYAN = "\u001b[36m"
+const ANSI_WHITE = "\u001b[37m"
+const ANSI_RESET = "\u001b[0m"
+
+const ANSI_ALT_SCREEN = "\x1b[?1049h"
+const ANSI_EXIT_ALT_SCREEN = "\x1b[?1049l"
```
Comment (d3flex) [inline cmd/openqa-mon/tui.go:28]: what do you think to make those also types rather than constants? ttycolors & ttystatus or something similar?

**Review by d3flex — COMMENTED** [review]
I would suggest to add a few tests. Especially in cases like this, it would be convenient to catch regressions while you are moving parts around

**Comment by grisu48** [comment]
> I would suggest to add a few tests. Especially in cases like this, it would be convenient to catch regressions while you are moving parts around

Good point! There is https://github.com/grisu48/openqa-mon/issues/28 already for it and it partly exists already but we're not there yet. It will come though.

File: cmd/openqa-mon/tui.go (hunk 1 above)
Comment (grisu48) [reply cmd/openqa-mon/tui.go:28]: What do you mean with types? Can you provide an example?

**Pull request merged**